crates/task-impls/src/consensus.rs

```diff
@@ -361,16 +361,16 @@ where
 }
 
 impl<
-        TYPES: NodeType<BlockHeader = H, BlockPayload = VIDBlockPayload>,
+        TYPES: NodeType<BlockPayload = VIDBlockPayload>,
         I: NodeImplementation<
             TYPES,
             Leaf = SequencingLeaf<TYPES>,
             ConsensusMessage = SequencingMessage<TYPES, I>,
         >,
         A: SequencingConsensusApi<TYPES, SequencingLeaf<TYPES>, I> + 'static,
-        H: BlockHeader<Payload = VIDBlockPayload>,
     > SequencingConsensusTaskState<TYPES, I, A>
 where
+    TYPES::BlockHeader: BlockHeader<Payload = VIDBlockPayload>,
```

Comment:
`NodeType` also has `type BlockPayload`, which may not be `VIDBlockPayload`. This brings up the question of whether we can make the payload a struct rather than a trait (issue https://github.com/EspressoSystems/HotShot/issues/1815). I'm going to discuss this with the team today.

Reply:
Actually, let me add this and the above comments to the issue https://github.com/EspressoSystems/HotShot/issues/1815, and I'll work on it after merging this PR.